contrib/opencensus-ext-azure/opencensus/ext/azure/common/protocol.py

```diff
@@ -128,6 +128,29 @@ def __init__(self, *args, **kwargs):
         self.ver = self.ver
         self.message = self.message
 
+class MetricData(BaseObject):
+    _default = BaseObject(
+        ver=2,
+        metrics=[],
+        properties=None,
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(MetricData, self).__init__(*args, **kwargs)
+        self.ver = self.ver
+        self.metrics = self.metrics
+
+class DataPoint(BaseObject):
+    _default = BaseObject(
+        name='',
+        value=0.0,
```

Comment:
Should have all the fields defined according to https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/DataPoint.bond.